ui/src/translations/en.json

```diff
@@ -96,6 +96,7 @@
   "no_storage_class_found": "No storage class found.",
   "storage_class_is_required": "A storage class is required in order to create a volume. Please create one in Kubernetes.",
   "learn_more": "Learn more",
+  "solution": "Solution",
```

Comment:
Don't we have something to handle the pluralization in the framework we use fo the translation?